sechub-sereco/src/test/java/com/daimler/sechub/sereco/importer/NetsparkerHtmlToAsciiDocConverterTest.java

```diff
@@ -0,0 +1,163 @@
+// SPDX-License-Identifier: MIT
+package com.daimler.sechub.sereco.importer;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class NetsparkerHtmlToAsciiDocConverterTest {
+
+    @Test
+    public void single_paragragph() {
+        /* prepare */
+        String html = "<p>This is text.</p>\n";
+        String expected = "This is text.";
+        
+        /* execute */
+        String actual = NetsparkerHtmlToAsciiDocConverter.convert(html);
+
+        /*test */
+        assertEquals(expected, actual);
+    }
+    
+    @Test
+    public void many_paragragphs() {
+        /* prepare */
+        String html = "<p>First paragraph.</p>\n<p>Second paragraph.</p>\n<p>Third paragraph.</p>\n";
+        String expected = "First paragraph.\n \nSecond paragraph.\n \nThird paragraph.";
```

Comment:
This whitespace between ´\n \n´ is kind of strange. I also checked it and could not find a reason for it.